packages/core/types/zapier.generated.d.ts

```diff
@@ -323,6 +333,12 @@ export interface Field {
    * "https://{{input}}.yourdomain.com").
    */
   inputFormat?: string;
+
+  /**
+   * Allows for additional metadata to be stored on the field.
+   * Supports simple key-values only (no sub-objects or arrays).
+   */
+  meta?: FieldChoices;
```

Comment:
assuming this should be `FieldMeta` instead of `FieldChoices`?

Reply:
Yes, good call! I am not sure why the tooling keeps generating `FieldChoices` 🤔 will take a look!

Reply:
Oh, it was an obvious typo 🤦🏻 Fixed!